g2p/mappings/utils.py

```diff
@@ -131,21 +142,22 @@ def normalize(inp: str, norm_form: str):
 
     Also, find any Unicode Escapes & decode 'em!
     """
-    if norm_form not in ["none", "NFC", "NFD", "NFKC", "NFKD"]:
```

Comment:
It seems like `None` was a valid value below, but we don't include it here, so that check was meaningless.  Made it meaningful...